setup.py

```diff
@@ -0,0 +1,62 @@
+"""setuptools based setup module.
+
+Adapted from https://github.com/pypa/sampleproject/blob/main/setup.py
+"""
+
+# Always prefer setuptools over distutils
+from setuptools import setup, find_packages
+import pathlib
+import versioneer
+
+here = pathlib.Path(__file__).parent.resolve()
+
+# Get the long description from the README file
+long_description = (here / "README.md").read_text(encoding="utf-8")
```

Comment:
That's neat

Reply:
Yep! I hadn't used `pathlib` either (this is coming from the pypa template linked in the description)

Reply:
Just be aware that pathlib creates objects and not strings. Lots of things will barf giving a pathlib path rather than a string path